peercrawler.py

```diff
@@ -495,6 +553,10 @@ def main():
     else:
         verbosity = args.verbosity if (args.verbosity is not None) else 1
         peerman = peer_manager(ctx, verbosity=verbosity)
+
```

Comment:
The starting of the background threads should be done inside the peer manager.
It is an implementation feature of the peer manager.
The html and rest peer manager scripts also need listening mode.